lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -48,8 +49,8 @@ class Inclusion:
                          'N': Inclusion.NOT,
                          'F': Inclusion.FILTER}
 
-INCLUDE_SECTION_PATTERN = re.compile('chapter|sequential')
-INCLUDE_PROBLEM_PATTERN = re.compile('problem')
+INCLUDE_SECTION_PATTERN = re.compile('|'.join(settings.INSTRUCTOR_QUERY_SECTION_TYPES))
```

Comment:
@dcadams @sigberto Sorry, my comment was referencing the `chapter|sequential` bit.
I agree problem types should be customized.

@dcadams Grab me in person if we're still on a different page.